### PR TITLE
Completed the code challenge, all 11 tests passed.

### PR DESCRIPTION
**Pull request by dannili**
**Overview**
A mini python project that allows users to login, signup, add tacos, and log out.

Refer to [Build a Social Network Sith Flask](https://teamtreehouse.com/library/build-a-social-network-with-flask/tacocat-challenge/the-challenge)

**Acceptance Criteria**
- [x] User accounts have an email address and a password
- [x] User email addresses are unique
- [x] Passwords are hashed
- [x] Logged in users see menu items for log out and to create a taco
- [x] Logged out users see menu items to log in or sign up
- [x] Tacos have a protein, a shell, a true/false for cheese, and a freeform area for extras
- [x] Once entered, tacos show up on the home page
- [x] If there aren't any tacos, show a message on the home page

**Testing**
* Unit tests

**Screenshot**
screen shot 2018-06-27 at 11 07 47 am

### CODE REVIEW

File: forms.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            email_exists
+        ])
+    password = PasswordField(
+        'Password',
+        validators=[
+            DataRequired(),
+            Length(min=8),
+            EqualTo('password2', message='Passwords must match.')
+        ])
+    password2 = PasswordField(
```
Comment (TeleCrichard) [inline forms.py]: as @tabiodun suggested on my Repo, you may want to consider renaming this variable to something more descriptive like `confirm_password`.

**Review by TeleCrichard — APPROVED** [review]
Other than the below suggestion, this looks great Danni!

**Comment by maxlvl** [comment]
👍 looks great! Nice job.

**Review by maxlvl — APPROVED** [review]

**Comment by dannili** [comment]
Thanks guys! Merging the PR.

**Pull request merged**